packages/react-ui-components/src/Button/style.css

```diff
@@ -36,10 +36,12 @@
     padding: 0 8px;
 }
 
-.btn--clean,
-.btn--clean i {
+.btn--clean {
     background-color: var(--brandColorsContrastDarker);
 }
+.btn--clean i {
+    background-color: 0;
```

Comment:
You are right :+1: 
`background-color: 0` is invalid
transparent or something like rgba(0, 0, 0, 0) is more what I was looking for. I will add this later